### PR TITLE
prompt_toolkit: update to 2.0.9

### PR DESCRIPTION
**Pull request by xaltsc**
Closes #9232

### CODE REVIEW

**Comment by maxice8** [comment]
remove the merge commit by rebasing

**Comment by maxice8** [comment]
please apply same fixes i pointed out in the Pull request you got the contents from.

**Comment by maxice8** [comment]
When applying them use git commit --amend, you can add

```
Co-authored-by: git-name <git-email>
```

to the end of the commit, to receive credit for the changes.

**Comment by xaltsc** [comment]
> please apply same fixes i pointed out in the Pull request you got the contents from.

I did that, didn't I ?

File: srcpkgs/python-jupyter_console/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+pycompile_module="jupyter_console"
 hostmakedepends="python-setuptools python3-setuptools"
 depends="python-ipython python-ipython_ipykernel python-jupyter_client
- python-prompt_toolkit python-Pygments"
-pycompile_module="jupyter_console"
+ python-prompt_toolkit1 python-Pygments"
 short_desc="Jupyter terminal console (Python2)"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
-homepage="https://github.com/jupyter/jupyter_console"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/python-jupyter_console/template]: ^BSD-3-Clause

**Comment by xaltsc** [comment]
lolptdr

**Comment by maxice8** [comment]
Each individial package need it's own commit

**Comment by maxice8** [comment]
LGTM

**Comment by CameronNemo** [comment]
#10774 this is a bunk transition @xaltsc thank you for your contribution, but please keep the old name with the old version. the new (incompatible) version needs a new package name. `python-prompt_toolkit2` in this case.

**Pull request merged**